lib/browser/api/web-contents.ts

```diff
@@ -417,11 +417,7 @@ WebContents.prototype.loadFile = function (filePath, options = {}) {
   }));
 };
 
-WebContents.prototype.loadURL = function (url, options) {
-  if (!options) {
-    options = {};
-  }
-
+WebContents.prototype.loadURL = function (url, options = {}) {
```

Comment:
Instead of making this a default param, we could instead leave this as it was and instead change line 487 to use nullish coalescing?

```js
this._loadURL(url, options ?? {});
```

Reply:
good idea :) done